code/datums/components/butchering.dm

```diff
@@ -92,7 +94,8 @@
  * - [butcher][/mob/living]: The mob doing the butchering
  * - [meat][/mob/living]: The mob being butchered
  */
-/datum/component/butchering/proc/Butcher(mob/living/butcher, mob/living/meat)
+/datum/component/butchering/proc/on_butchering(mob/living/butcher, mob/living/meat)
+	SIGNAL_HANDLER
```

Comment:
These are called in 3 different ways and two of them aren't by signals.

The Signal can be a different proc (like ``on_drill``) that has ``SIGNAL_HANDLER`` and calls *this* one Async using something like
``INVOKE_ASYNC(src, .proc/on_butchering, ...)``

It's important because signals pass themselves in the args by default, which we usually set as either ``atom/source`` or ``datum/source``

Reply:
ahh i see makes sense on it

Reply:
alright done